operator/v1/types_console.go

```diff
@@ -143,8 +143,97 @@ type Capability struct {
 }
 
+// ThemeType is the value of the CustomLogo Theme Type field that determines the theme mode in the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// ThemeTypeDark represents the dark theme for the console UI.
+	ThemeTypeDark ThemeType = "Dark"
+
+	// ThemeTypeLight represents the light theme for the console UI.
+	ThemeTypeLight ThemeType = "Light"
+
+	// ThemeTypeDefault represents the default theme for all unspecified themes.
+	ThemeTypeDefault ThemeType = "Default"
+)
+
+// LogoType is the value of the CustomLogo Type field that determines if the logo is for the masthead or the favicon in the console UI.
+type LogoType string
+
+const (
+	// Masthead represents the logo in the masthead.
+	LogoTypeMasthead LogoType = "Masthead"
+
+	// Favicon represents the favicon logo.
+	LogoTypeFavicon LogoType = "Favicon"
+)
+
+// Theme defines a theme type for the console UI.
+type Theme struct {
+	// type is a required field that specifies the type of the logo for the console UI.
+	// This field is an enum with valid values "Dark", "Light" and "Default".
+	// The type determines whether the logo is for the dark, light or default theme.
+	// When set to "Dark", the corresponding logo for the dark theme will be used in the console UI.
+	// When set to "Light", the corresponding logo for the light theme will be used in the console UI.
+	// When set to "Default", the corresponding logo will be used for any unspecified theme(s) in the console UI.
+	// If only the dark or light theme is specified, the default OCP logo will be used for the unspecified theme.
+	// +kubebuilder:validation:Enum="Dark";"Light";"Default"
+	// +required
+	Type ThemeType `json:"type"`
+
+	// file is a required field that references a specific file within a ConfigMap in the openshift-config namespace.
+	// This field allows the console to locate and use the specified file containing a custom logo.
+	// You can create it with a command like:
+	// - 'oc create configmap custom-logos-config --namespace=openshift-config --from-file=/path/to/file'
+	// The ConfigMap key must include the file extension for the dark or light or default theme so that the console serves the file with the correct MIME type.
+	// Recommended Masthead and About Modal logo specifications:
+	// Image size must be less than 1 MB due to constraints on the ConfigMap size.
+	// Dimensions: Max height of 60px and max width of 200px.
+	// SVG format preferred.
+	// Recommended Favicon specifications:
+	// Dimensions: Max height of 16px and max width of 16px
+	// PNG format preferred.
+	// +required
+	File configv1.ConfigMapFileReference `json:"file"`
+}
+
+// CustomLogo defines a configuration based on theme types for the console UI logo.
+type CustomLogo struct {
+	// type is a required field that specifies the type of the logo for the console UI. It determines whether the logo is for the masthead or favicon.
+	// Valid values are "Masthead" and "Favicon".
```

Comment:
No need to quote the allowed values

Reply:
You could just say `Valid values are Masthead and Favicon` (no quotes surrounding the allowed valued)